utils/wesnoth-map-diff/src/images.ts

```diff
@@ -36,7 +38,11 @@ const readTerrainImages = async () => {
   const promises = Object
     .entries(dictTerrainType2ImagesPath)
     .map(async ([terrainType, imageName]) => {
-      const image = await Jimp.read(`${imageBasepath}/${imageName}.png`)
+      let image = await Jimp.read(`${imageBasepath}/${imageName}.png`)
+      if (!image) {
+        console.log(`"${imageName}" not found in "${imageBasepath}"; falling back to a sensible default`)
+        image = await Jimp.read(`../../../data/core/images/misc/blank-hex.png`)
+      }
```

Comment:
Did you test it?
I tried now, and this fallback isn't working.

```
/Users/macabeus/ApenasMeu/wesnoth/utils/wesnoth-map-diff/build/images.js:62
                throw new Error(`Missing image for "${baseCode}"`);
                      ^

Error: Missing image for "AA"
```

If you want to use a fallback image if it wasn't found, we should update the `getTile` function on `wesnoth-map-diff/src/images.ts`

Reply:
Even if we return the code removed on https://github.com/wesnoth/wesnoth/pull/7499/commits/e4c6a36a176041e93c8bc0ae6e6dafd1a8556e45 the fallback isn't working btw

Reply:
I applied your patch from the other conversation in [c933a7d](https://github.com/wesnoth/wesnoth/pull/7499/commits/c933a7d937f586481520b18bb98f20aa77522a87); assuming that that fixes this